tests/Filter/StringFilterTest.php

```diff
@@ -34,6 +34,18 @@ public function testEmpty()
         $this->assertFalse($filter->isActive());
     }
 
+    public function testNullValue()
```

Comment:
Added testCase.

When the filter is passed with a null value it should be inactive.